### DISCUSSION TITLE
Lập trình hướng sự kiện (event driven) có gì khác với các kiểu lập trình khác

### DISCUSSION THREAD

**Discussion by ghost** [codingreshapefuture/Community#19] [Q&A] [answered]
Lập trình hướng sự kiện (event driven) có gì khác với các kiểu lập trình khác

**Comment by lucthienphong1120** [comment, marked as answer]
Trong lập trình máy tính, lập trình hướng sự kiện (tiếng Anh: Event-driven programming) là một mô hình lập trình trong đó luồng của chương trình được xác định bởi các sự kiện như hành động người dùng (nhấp chuột, bấm phím), đầu ra cảm biến, hoặc thông điệp từ các chương trình hoặc các luồng.
